aries-site/src/data/wcag/radiobuttongroup.json

```diff
@@ -0,0 +1,445 @@
+  },
+  {
+    "rule": "2.1.3",
+    "label": "Keyboard (No Exception)",
+    "status": "passed"
+  },
+  {
+    "rule": "2.1.4",
+    "label": "Character Key Shortcuts",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.2.1",
+    "label": "Timing Adjustable",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.2.2",
+    "label": "Pause, Stop, Hide",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.2.3",
+    "label": "No Timing",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.2.4",
+    "label": "Interruptions",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.2.5",
+    "label": "Re-authenticating",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.2.6",
+    "label": "Timeouts",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.3.1",
+    "label": "Three Flashes or Below Threshold",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.3.2",
+    "label": "Three Flashes",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.3.3",
+    "label": "Animation from Interactions",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.1",
+    "label": "Bypass Blocks",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.2",
+    "label": "Page Titled",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.3",
+    "label": "Focus Order",
+    "status": "passed"
+  },
+  {
+    "rule": "2.4.4",
+    "label": "Link Purpose (In Context)",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.5",
+    "label": "Multiple Ways",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.6",
+    "label": "Headings and Labels",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.7",
+    "label": "Focus Visible",
+    "status": "passed"
+  },
+  {
+    "rule": "2.4.8",
+    "label": "Location",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.9",
+    "label": "Link Purpose (Link Only)",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.10",
+    "label": "Section Headings",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.4.11",
+    "label": "Focus Not Obscured (Minimum)",
+    "status": "passed"
+  },
+  {
+    "rule": "2.4.12",
+    "label": "Focus Not Obscured (Enhanced)",
+    "status": "passed"
+  },
+  {
+    "rule": "2.4.13",
+    "label": "Focus Appearance",
+    "status": "passed"
+  },
+  {
+    "rule": "2.5.1",
+    "label": "Pointer Gestures",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.5.2",
+    "label": "Pointer Cancellation",
+    "status": "passed",
+    "notes": "No down event."
+  },
+  {
+    "rule": "2.5.3",
+    "label": "Label in Name",
+    "status": "conditional",
+    "notes": "Implementer needs to ensure the ARIA label is informative to the user."
+  },
+  {
+    "rule": "2.5.4",
+    "label": "Motion Actuation",
+    "status": "not-applicable"
+  },
+  {
+    "rule": "2.5.5",
+    "label": "Target Size (Enhanced)",
+    "status": "needs discussion",
```

Comment:
@jcfilben How do you judge this status?

Reply:
I think this should be marked as failing because the touch target height of a radiobutton control isn't 44px